go-apps/meep-metrics-engine/api/v2/swagger.yaml

```diff
@@ -426,6 +451,89 @@ definitions:
     example:
       time: "2019-11-24T12:45:00-5:00"
       mermaid: "Alice ->> Bob: Hello!"
+  DataflowQueryParams:
+    type: "object"
+    properties:
+      tags:
+        type: "array"
+        description: "Tag names to match in query. Supported values:<br> "
+        items:
+          $ref: "#/definitions/Tag"
+      scope:
+        $ref: "#/definitions/Scope"
+      fields:
+        type: "array"
+        description: "Requested information. Supported values:<br> "
```

Comment:
Add list of supported fields + update example accordingly.

Reply:
Addressed.